ajax/libs/colors/3.0.0/js/colors.js

```diff
@@ -0,0 +1,27 @@
+(function() {
```

Comment:
Yes, minified version is already included at https://github.com/samme/cdnjs/blob/95900c27e8b863edef7116b43244708cbe8217b2/ajax/libs/colors/3.0.0/js/colors.min.js